channeldb/invoices.go

```diff
@@ -2761,6 +2761,107 @@ func delAMPSettleIndex(invoiceNum []byte, invoices,
 	return nil
 }
 
+// DeleteCanceledInvoices deletes all canceled invoices from the database.
+func (d *DB) DeleteCanceledInvoices(_ context.Context) error {
+	return kvdb.Update(d, func(tx kvdb.RwTx) error {
+		invoices := tx.ReadWriteBucket(invoiceBucket)
+		if invoices == nil {
+			return invpkg.ErrNoInvoicesCreated
```

Comment:
same here with returning nil rather (and for index below). I think currently if there are no invoices in the DB, then the invoice registry will fail to start if `GcCanceledInvoicesOnStartup` is true

Reply:
done :)